### DISCUSSION TITLE
Upper case issue with some languages

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#21822] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/50368878/209711778-3a4075ad-701e-4fd6-8de9-deaab693806d.png)
In the language section the french and spanish does not have a upper case first letter and i didnt find on crowdin where i can fix that

**Comment by bdach** [comment, marked as answer]
we've been through this before once.

https://github.com/ppy/osu-web/issues/6840
